### ISSUE TITLE
Some errors occurred during installation. steamclient.so': No such file or directory

### ISSUE THREAD

**Issue by SyaoranChang** [nitrog0d/palworld-arm64#1]
Hi,

I am also using Oracle Cloud free tier
Os is Oracle Linux 8.9.
I run docker compose up, but it doesn't seem to have finished properly.
Is there a step I'm missing?
Do I need to install SteamCMD? If so, what OS version should I install?
Could you help me?

[+] Running 11/11
 ✔ palworld-server 10 layers [⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿]      0B/0B      Pulled                                                                           163.3s 
   ✔ ce9ebea987c2 Pull complete                                                                                                              0.8s 
   ✔ 49176cc8e18c Pull complete                                                                                                              2.3s 
   ✔ 6159ddd6f1a6 Pull complete                                                                                                              0.9s 
   ✔ 91d1ffe1168d Pull complete                                                                                                              1.5s 
   ✔ 5eb40751c7d5 Pull complete                                                                                                              2.1s 
   ✔ 8ebb2aa73f4e Pull complete                                                                                                              2.2s 
   ✔ 517d95f5eab6 Pull complete                                                                                                             20.4s 
   ✔ e4f8b9375297 Pull complete                                                                                                              2.8s 
   ✔ a0a86f146452 Pull complete                                                                                                              4.1s 
   ✔ 48c2a918aa48 Pull complete                                                                                                              3.3s 
[+] Running 2/1
 ✔ Network palworld-arm64_default  Created                                                                                                   0.2s 
 ✔ Container palworld-server       Created                                                                                                   0.0s 
Attaching to palworld-server
palworld-server  | Server not found! Installing... (Do not panic if it looks stuck)
palworld-server  | <jemalloc>: Unsupported system page size
palworld-server  | /home/steam/init-server.sh: line 4:     8 Segmentation fault      (core dumped) FEXBash './steamcmd.sh +force_install_dir "/palworld" +login anonymous +app_update 2394010 validate +quit'
palworld-server  | Checking for updates... (Do not panic if it looks stuck)
palworld-server  | <jemalloc>: Unsupported system page size
palworld-server  | /home/steam/init-server.sh: line 4:     9 Segmentation fault      (core dumped) FEXBash './steamcmd.sh +force_install_dir "/palworld" +login anonymous +app_update 2394010 validate +quit'
palworld-server  | cp: cannot stat '/home/steam/Steam/linux64/steamclient.so': No such file or directory
palworld-server  | Starting server... You can safely ignore Steam errors! (Also the server has pretty much 0 logging, so just try connecting to it)
palworld-server  | <jemalloc>: Unsupported system page size
palworld-server  | /home/steam/init-server.sh: line 8:    12 Segmentation fault      (core dumped) FEXBash "./PalServer.sh $START_OPTIONS"
palworld-server exited with code 0

**Comment by nitrog0d** [comment]
I'm using Ubuntu 22.04 as host on my oracle arm64 server. Using 24gb of ram and 4 cores, works fine for me, not sure what "Unsupported system page size" means?

**Comment by santowyo** [comment]
did you open any other ports?

**Comment by nitrog0d** [comment]
If you're asking me, Docker opens ports for me automatically, however you also need to open the default port on Oracle Cloud VPC stuff

**Comment by BrunoDomenesDutra** [comment]
![image](https://github.com/nitrog0d/palworld-arm64/assets/9719031/026bf080-9009-4766-805e-73b12ddf7806)

Everything working fine here.

N00b question @nitrog0d, is there a command to safe turn off the server?

**Comment by nitrog0d** [comment]
In game you can do /AdminPassword <password> and then /Save to save the world, there's also /DoExit that closes the server (but doesn't stop the docker container, the server will restart)

**Comment by SyaoranChang** [comment]
Thank you for your reply.
I'll see what I can do about the Unsupported system page size issue.

My Oracle Cloud free tier status
![Snipaste_2024-01-25_11-55-05](https://github.com/nitrog0d/palworld-arm64/assets/18550228/425d790a-e21b-4530-92f9-9f15f8155496)

**Comment by SyaoranChang** [comment]
It seems to be because FEX cannot generally support 16K pages?
https://github.com/FEX-Emu/FEX/issues/1650#issuecomment-1416922823

4k page emulation
https://github.com/FEX-Emu/FEX/issues/1921

Maybe it would be better if I switched to Ubuntu?

**Comment by nitrog0d** [comment]
Yes, I built it in Ubuntu and FEX says it has compatibility with Ubuntu by default?

**Comment by Deses** [comment]
What ubuntu do you use? The aarch64 one, to be able to use the Ampere cpus, right?

**Comment by YenyulLover** [comment]
I am facing the same issue on my own Oracle instance. Did anyone figure out a solution to this?

**Comment by paulwer** [comment]
has anyone build it for arm64 Oracle Linux 8.9?
